shell/platform/android/test/io/flutter/plugin/editing/InputConnectionAdaptorTest.java

```diff
@@ -1028,169 +1028,6 @@ public void testCursorAnchorInfo() {
     assertNull(testImm.lastCursorAnchorInfo);
   }
 
-  @Test
```

Comment:
Should we add a test to verify that KeyEvent.KEYCODE_DEL doesn't do anything?

Reply:
OK, I'll add that. Should be a simple matter of reversing the expects in the original test.